tests/integration/blueprints/datamanager/test_datamanager_integration.py

```
@@ -383,3 +383,173 @@ def test_float_entity_id_matches_platform_integer(self):
+                    ):
+                        with patch(
+                            "application.blueprints.datamanager.controllers.transform.get_entity_count_for_organisation_and_dataset",
+                            return_value=600,
+                        ):
+                            with patch(
+                                "application.blueprints.datamanager.controllers.transform.get_entities_for_organisation_and_dataset",
+                                return_value=platform_entities,
+                            ):
+                                resp1 = client.get("/datamanager/check-transform/test-id?entity_page=1")
+                                resp2 = client.get("/datamanager/check-transform/test-id?entity_page=2")
+        # page 1: 10 resource + 490 platform-only (8000000000–8000000489); page 2: remaining 110
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)